wiki/core/markdown/__init__.py

```diff
@@ -22,7 +22,7 @@ def core_extensions(self):
 
     def get_markdown_extensions(self):
         kwargs = settings.MARKDOWN_KWARGS
-        extensions = kwargs.get('extensions', [])
+        extensions = list(kwargs.get('extensions', []))
```

Comment:
No, `list()` is used to make a copy of the extension list. 
Basically done to avoid modifying the original markdown extension list.